### ISSUE TITLE
Missing VERSION file in repository causes error when building from source

### ISSUE THREAD

**Issue by takuy** [TheCellule/python-bleson#50]
```
C:\>pip install bleson
Collecting bleson
  Using cached https://files.pythonhosted.org/packages/ca/dd/21d00234512fed0a53c968e20c7a5aeb16d580e92de8deb25b337a954203/bleson-0.0.10.tar.gz
Collecting blesonwin (from bleson)
  Using cached https://files.pythonhosted.org/packages/f7/c3/6ee3360cbfe7883749ea0a7744017975caec626a82a8db510375eedd0014/blesonwin-0.0.11.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\me\AppData\Local\Temp\pip-install-u_rqh0wl\blesonwin\setup.py", line 12, in <module>
        version_file = open('VERSION')
    FileNotFoundError: [Errno 2] No such file or directory: 'VERSION'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\me\AppData\Local\Temp\pip-install-u_rqh0wl\blesonwin\
```

Installing on Windows 10, per docs. I get the error above. Any ideas?

pip 18.1
Python 3.7.2

**Comment by WayneKeenan** [comment]
The underlying issue is I've not checked in a file... 

However, a workaround would be to install Python 3.6 as pre-compiled packages (Wheels) are available.

**Comment by takuy** [comment]
I realized after posting that it installs fine with Python 3.6:
``pip3.6 install bleson ``

Thanks!

**Comment by WayneKeenan** [comment]
closing and moving to https://github.com/TheCellule/python-blesonwin/issues/4

**Comment by WayneKeenan** [comment]
this was fixed today for sdist installs via commit 30e3bd6

**Comment by khazhyk** [comment]
I don't think that fixed it, I think you need to add the file to the blesonwin manifest, not bleson

```sh
 Using cached https://files.pythonhosted.org/packages/f7/c3/6ee3360cbfe7883749ea0a7744017975caec626a82a8db510375eedd0014/blesonwin-0.0.11.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\khazhy\AppData\Local\Temp\pip-install-5q5fxlbp\blesonwin\setup.py", line 12, in <module>
        version_file = open('VERSION')
    FileNotFoundError: [Errno 2] No such file or directory: 'VERSION'
```
